tests/components/ct-svelte-vite/package.json

```diff
@@ -18,6 +18,6 @@
     "vite": "^5.2.8"
   },
   "dependencies": {
-    "svelte": "^4.2.8"
+    "svelte": "^5.19.9"
```

Comment:
I think we actually want to introduce ct-svelte4-vite to keep testing for vite4, wdyt @agg23

Reply:
Agreed

Reply:
Sure, I'll will try to do that today. Thanks for the input.